README.md

```diff
@@ -1,33 +1,40 @@
 # Welcome
 
-This repository contains the source code for:
+This repository contains the source code & documentation for:
 
 * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
 * The Windows console host (`conhost.exe`)
 * Components shared between the two projects
 * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
 * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
+
 Other related repositories include:
 
 * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
 
 ## Installing & running Windows Terminal
 
-> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
+### Operating System Support
+
+Windows Terminal can be installed on:
 
-### From the Microsoft Store
+* Windows 10 1903 (build 18362) or later
+* [Windows Server 1903](https://docs.microsoft.com/en-us/windows-server/get-started-19/whats-new-in-windows-server-1903) or later (install manually - see "From this repository" below)
 
-Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated:
+### Install via the Microsoft Store
+
+On Windows 10, we recommend downloading the Windows Terminal from the Microsoft Store and it'll be continuously updated:
 
 <a href='//www.microsoft.com/store/apps/9n0dx20hk701?cid=storebadge&ocid=badge'><img src='https://assets.windowsphone.com/85864462-9c82-451e-9355-a3d5f874397a/English_get-it-from-MS_InvariantCulture_Default.png' alt='English badge' width="284" height="104" style='width: 284px; height: 104px;'/></a>
 
-### From this repository
-Every public release of Windows Terminal is avaiable for manual download from [this repo's Releases page](https://github.com/microsoft/terminal/releases)
+### Manually install builds from this repository
+For users who're unable to install Terminal from the Microsoft Store (e.g. on [Windows Server 1903](https://docs.microsoft.com/en-us/windows-server/get-started-19/whats-new-in-windows-server-1903) or later), Terminal builds can be manually downloaded from [this repo's Releases page](https://github.com/microsoft/terminal/releases).
 
-> ⚠ Note: If you manually install Terminal from [releases](https://github.com/microsoft/terminal/releases) they won't auto-update so be sure to regularly install the latest Terminal release to receive all the latest fixes and improvements!
+> ⚠ Note:
+> 1. If you install Terminal manually, be sure to install [the Desktop Bridge VC++ v14 Redist Package](https://www.microsoft.com/en-us/download/details.aspx?id=53175) otherwise Terminal may not install and/or run and may crash at startup
```

Comment:
```suggestion
> 1. If you install Terminal manually, be sure to install [the Desktop Bridge VC++ v14 Redistributable Package](https://www.microsoft.com/en-us/download/details.aspx?id=53175) otherwise Terminal may not install and/or run and may crash at startup
```